### PR TITLE
Homework #1 - Sumer Vaid

### PR DESCRIPTION
**Pull request by sumervaid**
Homework #1 - Sumer Vaid

### CODE REVIEW

**Comment by cwfowler** [comment]
I was not requested to review anyone's assignment so I am randomly choosing two as was instructed. Moving on, I think that Sumer did a great job on this assignment, especially since he avoided the common error that has been tripping up a lot of the rest of in the class of not putting a space between the hashtags and the text in order to create a header. He seems to have completed all of the steps indicated for this assignment to include: headers, lists, italicized and bolded text. The only thing that I would say is that the write-up possibly should have been included as a comment after the commit to Github had been made and not in the Markdown file but I myself would not have known to do so unless it been mentioned to me so I hesitate to actually criticize such. Great job! 

| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |        x      |            |
| **Ease of access**          |      x     |              |            |

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+(I have embedded the the Amazon-link into the title of the book)
+
+* [*Black Edge: Inside Information, Dirty Money, and the Quest to Bring Down the Most Wanted Man on Wall Street*](https://amzn.to/2Gsyo0F) by Sheelah Kolhatkar
+
+* [*The Gene: An Intimate History*](https://amzn.to/2EcnslC) by Siddhartha Mukherjee
+
+* [*Englightenment Now: The Case for Reason, Science, Humanism and Progress*](https://amzn.to/2H1Q4kJ) by Steven Pinker
```
Comment (cwfowler) [inline README.md]: As well, great touch on including the Amazon links for the books! Definitely, an interesting way to include the required link!

Comment (cwfowler) [inline README.md:10]: As well, great touch on including the Amazon links for the books! Definitely, an interesting way to include the required link!

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for Sumer Vaid
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x     |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |     x      |              |            |

Good job using many different techniques successfully in R Markdown (including merging multiple ones in the bullet points).  Your reflection is also thorough.  The one suggestion is that you could have added an image as well.

**Comment by estech01** [comment]
Evaluation HW-01 for Sumer Vaid
Coding Style: Excellent
Coding Strategy: Excellent
Presentation: Excellent
Creativity: Satisfactory
Ease of Access: Excellent

This was well-organized and easy to read! I liked the Amazon links, and the way you used headers to break this up. It would have been nice to have included a picture!

**Pull request closed without merging**